frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/EditIconModal/EditIconModal.tsx

```
@@ -234,7 +234,7 @@ const EditIconModal = ({
     // Default to VPP icon if available, otherwise fall back to default icon
     const defaultPreviewUrl =
       previewInfo.currentIconUrl &&
-      !previewInfo.currentIconUrl.startsWith("/api/")
+        !previewInfo.currentIconUrl.startsWith("/api/")
```

ESLint

[error] Delete `··`

(prettier/prettier)